src/test/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/AdapterSDKFakeContainer.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.fileanalysissuite.adaptersdk.impls.jaxrs;
+
+import io.github.fileanalysissuite.adaptersdk.interfaces.extensibility.RepositoryAdapter;
+
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.glassfish.jersey.test.inmemory.InMemoryTestContainerFactory;
+
+public class AdapterSDKFakeContainer extends JerseyTest
+{
+    RepositoryAdapter fakeAdapter;
+
+    public AdapterSDKFakeContainer()
+    {
+        super(new InMemoryTestContainerFactory());
+    }
+
+    @Override
+    protected ResourceConfig configure()
+    {
+        fakeAdapter = new FakeRepositoryAdapter();
+        return new ResourceConfig().register(new AdapterApiImpl(fakeAdapter));
```

Comment:
You're calling a private method here.  Generally it's best to only call the public methods from the unit tests.

```suggestion
        return new ResourceConfig().registerInstances(AdapterSdk.wrap(fakeAdapter));
```

Reply:
This warning message is still being produced which seems like a known issue, should this be disabled in the logs
https://github.com/eclipse-ee4j/jersey/issues/3700
WARNING: A provider io.github.fileanalysissuite.adaptersdk.impls.jaxrs.AdapterApiImpl registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider io.github.fileanalysissuite.adaptersdk.impls.jaxrs.AdapterApiImpl will be ignored.